src/slack.js

```diff
@@ -9,16 +9,24 @@ const instance = new Slack(process.env.ACCESS_TOKEN);
 const uploadToS3 = async (key, metadata, tmpPath) => {
   const s3 = new AWS.S3();
 
-  return s3
-    .upload({
-      Bucket: process.env.S3_BUCKET,
-      Key: `emoji/${key}`,
-      // Body: writeStream,
-      Body: fs.createReadStream(tmpPath),
-      ContentType: metadata.file.mimetype,
-      ACL: 'public-read',
-    })
-    .promise();
+  let res;
+  try {
+    res = await s3
```

Comment:
Why not just return here rather than assigning it?

Reply:
leftover from debugging